addon/components/frost-button.js

```diff
@@ -100,7 +122,15 @@ export default Ember.Component.extend({
    * ['small', 'medium', 'large', 'extra-large']
    * @type {String}
    */
-  size: 'medium',
+  size: '',
```

Comment:
Got it. Well in that case, I guess we need the `#MAJOR#` bump then because there is existing code which probably relies on the defaults and that code will break with this release.